receiver/azuremonitorreceiver/README.md

```diff
@@ -145,3 +145,52 @@ receivers:
 ## Metrics
 
 Details about the metrics scraped by this receiver can be found in [Supported metrics with Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported). This receiver adds the prefix "azure_" to all scraped metrics.
+
+## Azure API calls summary
+At each collection interval, here are the different Azure API that can be called.
+It can be useful to know that, in order to configure the client permission in Azure or to choose the 
+right configuration based on your needs.
+
+### [Subscriptions - Get](https://learn.microsoft.com/en-us/rest/api/resources/subscriptions/get?view=rest-resources-2022-12-01&tabs=HTTP)
+```yaml
+conditions:
+- subscription_ids is set
+- discover_subscriptions is false or not set
+- resource_attributes.subscription.enabled is true
+cardinality: once per sub id
+```
+### [Subscriptions - List](https://learn.microsoft.com/en-us/rest/api/resources/subscriptions/list?view=rest-resources-2022-12-01&tabs=HTTP)
+```yaml
+conditions:
+- discover_subscriptions is true
+cardinality: once
```

Comment:
This is a paged API, right? So it may be more than once depending on the number of subscriptions. Same for other List endpoints.

Reply:
That's true yes, I updated with that precision. Unfortunately I didn't  find the page size in the API documentation or client codes. This is a mystery that Azure seems to keep secret deep in the backend ^^'